runtime/doc/options.txt

```diff
@@ -4086,7 +4086,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	listing continues until finished.
 
 						*'mouse'*
-'mouse'			string	(default "")
+'mouse'			string	(default "a")
```

Comment:
should probably be "nvi" so that one can copy during a press-enter prompt

Reply:
Done